openlitespeed/Dockerfile-server

```diff
@@ -0,0 +1,71 @@
+
+FROM ubuntu:focal as builder
+
+ENV TZ=Europe/Zurich
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Update apt software 
+RUN apt update
+
+# Install prerequisites
+RUN apt install git libz-dev libevent-dev cmake gcc ninja-build libunwind-dev pkg-config python3 python3-psutil golang-go -y
+
+WORKDIR /opt
+
+# Download openlitespeed repository
+RUN git clone https://github.com/litespeedtech/openlitespeed.git 
+
+COPY build.sh.patch /opt/
+COPY build_bssl.sh.patch /opt/
+
+# Apply patch to build.sh
+RUN cd openlitespeed && git apply --reject --whitespace=fix /opt/build.sh.patch 
+
+
+# Build and install openlitespeed
+# The webadmin password is set to password
+RUN cd openlitespeed && ./build.sh &&  sed 's/#OPENLSWS_PASSWORD=/OPENLSWS_PASSWORD=password/' ols.conf  -i && ./install.sh
+
+WORKDIR /root/
```

Comment:
Why is there a need to generate certificates (and keys!) and store them in the Docker image? Could this not be done at server startup (avoiding storage of conceptually sensitive material in the docker image)? If I read the client code right, it anyway downloads the CA.crt anew, so that cert/key pair could be generated at server startup, right?

Reply:
You are right. I have added a serverstart.sh file which generates the certificate and key files and starts the server.
Thanks!